app/models/user.rb

```diff
@@ -44,4 +47,27 @@ def forget
     def session_token
       remember_digest || remember
     end
+
+    # アカウントを有効にする
+    def activate
+      update_columns(activated: true, activated_at: Time.zone.now)
```

Comment:
nits
```suggestion
      update_columns(activated: true, activated_at: Time.current)
```

IMO
rails のメソッドである `Time.current` の方がよく用いられる印象です。